docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -150,6 +150,83 @@ To investigate further, you can view the details of the build captured as labels
+
+Storing Jenkins pipeline logs in Elastic requires:
+
+* Elastic {observability} version 8.1 or higher.
+* The OpenTelemetry Protocol endpoint configured on the Jenkins OpenTelemetry Plugin must be
+reachable from the Jenkins Agents (i.e. don't specify a localhost OTLP endpoint unless OpenTelemetry
+collectors are also deployed on the Jenkins Agents).
+* When using OpenTelemetry Collectors, set up a https://opentelemetry.io/docs/collector/configuration/#service[logs pipeline]
+in addition to the traces and metrics pipelines.
+
+To store pipeline logs in Elastic:
+
+1. Navigate to the _OpenTelemetry_ section of the Jenkins configuration screen.
+2. Set the _OTLP Endpoint_.
+3. Use the _Add Visualisation {observability} Backend_ drop-down to select the *Elastic {observability}* option.
+4. Set the _{kib} base URL_.
+5. Click the _Advanced_ button to choose a storage integration strategy.
+There are two options for storing pipeline logs in Elastic {observability}:
+    * *Store pipeline logs in Elastic and visualize logs both in Elastic and through Jenkins*,
+    which means you can view logs stored in Elastic on demand in the Jenkins UI.
+    Read more in <<ci-cd-visualize-logs-kibana-and-jenkins>> below.
+    * *Store pipeline logs in Elastic and visualize logs exclusively in Elastic*,
+    which means logs will no longer be visible through the Jenkins UI.
+    Read more in <<ci-cd-visualize-logs-kibana>> below.
+
+Visualizing logs both in Elastic and through Jenkins is recommended because it provides a more seamless user experience
+by continuing to render the logs in the Jenkins UI while allowing you to verify the {es} setup.
+
+[discrete]
+[[ci-cd-visualize-logs-kibana-and-jenkins]]
+=== Visualize logs in both {kib} and Jenkins
+
+The Jenkins OpenTelemetry Plugin also provides pipeline log storage in {es} while enabling you to
+either visualize the logs in {kib} or continue to display them through the Jenkins pipeline build console.
+
+[role="screenshot"]
+image::images/ci-cd-visualize-logs-kibana-and-jenkins-console.png[Jenkins Console Output page displaying both log contents and a link to view logs in Elastic {observability}, width=1000]
+
+This more advanced setup requires connecting the Jenkins Controller to {es} with read permissions
+on the `logs-apm.app` and preferably on the Metadata of the {ilm-init} policy of this index template
+(by default it's the `logs-apm.app_logs-default_policy policy`).
+Use "Validate {es} configuration" to verify the setup.
+
+.Architecture for storing pipeline logs in Elastic and visualizing logs in both Elastic and Jenkins
+image::images/ci-cd-visualize-logs-kibana-and-jenkins-architecture.png[Architecture diagram for storing pipeline logs in Elastic and visualizing logs in both Elastic and Jenkins, width=1000]
+
+[discrete]
+[[ci-cd-visualize-logs-kibana]]
+=== Visualize logs exclusively in {kib}
+
+The Jenkins OpenTelemetry Plugin provides turnkey pipeline log storage in {es} with visualization in {kib}.
```

Comment:
```suggestion
A simpler setup doesn't require to access {es} from the Jenkins Controller. In such case, the Jenkins pipeline build console no longer displays the logs but just an hyperlink to the {kib} logs visualization screen.
```

Reply:
How about something like this? 

```suggestion
Visualizing logs exclusively in {kib} involves a simpler setup that doesn't require access
to {es} from the Jenkins Controller. This is because the Jenkins pipeline build console
displays a hyperlink to the {kib} logs visualization screen instead of displaying the logs
in the Jenkins UI.
```